### ISSUE TITLE
It will not work

### ISSUE THREAD

**Issue by huba80** [condor2/Opencart_2302#16]
Hi,

Ive tried to use this patched version of Opencart. Now after spending weeks on it Ive realized this will never work unless you are a webprogramer and can fix your own problems. This patched version changes the Modification section within Opencart which will interfere with most third party Opencart extensions. This is a half in, half out solution where part of the Opencart is 2.3 other part is 3.0. When someone develops an extension they will make it for either v2.3 or v3.0. Some of my Extensions stopped working after patching my Opencart, some just simply disappeared from the menu. 
Lacking support is also a huge downside of this patch.

**Comment by condor2** [comment]
Almost all ocmod extensions should work just fine.

**Comment by IP-CAM** [comment]
Well, you're most likely just expecting too much, by 'globally' talking about 'Opencart'. Just take the 'Journal' Theme Extension as classicle Sample, the Extension Download is size/data-wise 'larger' than the OC Download itself, and it so fundamentally changes most of the 'default' OC Way of doing things. Other Downloads might be less 'comprehensive', but still change some basic 'OC' doings, and as a consequence of this, many other Extensions won't work in full, if not 'adapted', to meet some changes, done/made by other Mods. 

And never forget, OC comes for free, but that's all one can expect. And most OC Installations contain some 'Custom' Code, but OC-unfamiliar Users are just not aware of that, and expect most everything to just work as planned. So, better don't expect free Support, if one os not able, to fix all that by himself. Code Contributors exist, to generate Income with OC, because OC itself only serves that purpose, it's not a free Gift from heaven, to make/enable everybody, to get successful in Online Business. 

And 'latest' OC Releases are always widely untested Prototypes, OC has not the manpower, to test them out in full. That's, why they come for free too, it just needs to be understud. 


OR then, get one of the 'older' OC Versions, already known and proven to work, as they come by default.

**Comment by huba80** [comment]
This is why I will never use Journal Theme. I don't expect free support. In fact I didnt get a lot replies here. If someone here offers me paid support I might have gone with it. I actually just paid for one of my developer to fix some issues with the template etc.. 

Regardless the paid support... the developer may fix the problem with my 2 current extension... what happens then 6 months later I need a new extension and pay for support again as the patched version will not be compatible again. This is just simply not the way forward. Extension developers also decline support as this patched / hybrid Opencart will be out of their scope. 
I wanted to upgrade from 2.3.0.2 to Opencart v3. I met a bunch of issues, bugs therefore I decided to revert back to 2.3.0.2. This patch seemed tempting so I gave it a try. As I mentioned above a few (not all) extensions have not been compatible with this patch.
I don't complain of Opencart. I know it's free and it is difficult to maintain something in free time. One thing that I never liked in Opencart... It makes absolutely no sense coming out with new releases when there are many issues reported on the forum and they are being ignored. New versions come out without fixing long standing issues.  After 2-3 years of development Opencart v3 is still far from "stable". 
I hoped that this patched version would work for me but it just doesnt. I wasted 4-7 days to test it properly when I realized that it will not work. Anyone can take their time if they want and give it a try. Some extensions embed into main admin menu. For me Pavotheme and Nitrocache stopped working of my total 4 extensions.

To sum up this patched version would be great if the OCMOD part hadn't been changed. There is still future for 2.3.0.2 if someone fixes security and PHP compatibility issues.

**Comment by IP-CAM** [comment]
I wanted to upgrade from 2.3.0.2 to Opencart v3. I met a bunch of issues .........

Well, it depends, on how you tried to upgrade, your Attempt may had some influence on your Database, making it unable to use v.2.3.x Code again. But generally spoken, so-called UPGRADES would only function with a DEFAULT OC Installation, especially, if one moves up from One main Version OC to a higher Version Release anyway.

And it also depends on the PAVO Theme you're using, some of them also add bulkloads of Custom Code and Modules into Opencart. And the same applies for Nitrocache. But I am sure, that Condor2 would be able, to make all this work, one after one, if you care, to have a fully working v.2.3.x Installation.

Personally, I am just glad, I never tried, especialy not, because my 1.5.6.5 Release is still much simpler to handle, as secure as Fort Knox, and speedy like hell. My .htaccess already contains more than 3'700 blocked IP-Ranges, just to make sure, that all those, trying to hack it, never will be able to even call one of the Test Sites again ... :)

Good Luck!
Ernie

**Comment by condor2** [comment]
@huba80 You can use History function and download version with old OCMOD...

https://github.com/condor2/Opencart_2302/commits/master/upload
